micro/core/config.py

```diff
@@ -1,19 +1,22 @@
-import json
 import os
+import sys
+import json
 
 
 class Config:
-    def __init__(self):
-        self.__MICRO_CONFIG = "MICRO_CONFIG"
-        self.__path = os.environ.get(self.__MICRO_CONFIG, "")
+    def __init__(self, path=None):
         self.__conf = {}
-        if self.__path:
-            self.__load()
+        if path:
+            self.__load(path)
 
     def key(self, name):
         return self.__conf.get(name)
 
-    def __load(self):
-        with open(self.__path) as conf_file:
+    def __load(self, path):
+        if not os.path.exists(path) or os.path.isdir(path):
+            sys.stderr.write("ERROR: config file not found: {}".format(path))
+            sys.exit(1)
```

Comment:
A better way to do this is `sys.exit("ERROR: config file not found: {}".format(path))` see https://docs.python.org/3/library/sys.html#sys.exit

Reply:
Oh, and results in an exit code `1` and the text is sent to `stderr`..., so it is a much better way to do it, great! 👍

Reply:
solved in commit b2f08aba5dff0c10a7c217c824e0086e5472c060